app/components/LoginLocalStorage.js

```diff
@@ -60,6 +60,12 @@ let LoginLocalStorage = class LoginLocalStorage extends Component {
     }))
   }
 
+  handleKeyPress = (event) => {
+    if (event.key === 'Enter') {
```

Comment:
Please use constants.
We now have a `constants.js` under /core folder.
Can be something like:
```
export const KEYS = {
  ENTER: 'Enter',
}
```